packages/common-substrate/src/project/versioned/v0_2_0/model.ts

```diff
@@ -43,22 +43,40 @@ export class ProjectMappingV0_2_0 extends Mapping {
   file: string;
 }
 
+function validateObject(object: any, errorMessage = 'failed to validate object.'): void {
```

Comment:
This can be moved to `@subql/common`

Reply:
shall I move it to `@subql/common/project/versioned/v0_2_0/model.ts`  or otherwise move it to an utility file such as `@subql/common/project/utils.ts`?